packages/prism-theme-odyssey/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@okta/prism-theme-odyssey",
+  "version": "0.6.0",
+  "description": "A Prism.js theme built for Odyssey's documentation site.",
+  "private": true,
+  "author": "Okta, Inc.",
+  "license": "Apache-2.0",
+  "main": "src/theme.scss",
+  "scripts": {
+    "dev": "webpack-dev-server"
+  },
+  "dependencies": {
+    "@okta/odyssey": "^0.6.0",
+    "prismjs": "^1.14.0"
+  },
+  "devDependencies": {
+    "capture-website": "^1.1.0",
+    "html-webpack-plugin": "^4.3.0",
+    "sass": "^1.26.10",
+    "sass-loader": "^9.0.2",
+    "style-loader": "^1.2.1",
+    "webpack": "^4.43.0",
+    "webpack-cli": "^3.3.12",
+    "webpack-dev-server": "^3.11.0"
```

Comment:
There are no `yarn.lock` file changes here - please make sure to run `yarn install` and push up those changes before merging!

Reply:
Thank you!